tests/mock_environment/mock_environment.py

```diff
@@ -53,6 +53,7 @@
 os.environ["EC2_KEY_NAME"] = "test_ec2_key_name"
 os.environ["EMR_INSTANCE_PROFILE"] = "TEST_EMR_INSTANCE_PROFILE"
 os.environ["EMR_SERVICE_ROLE"] = "TEST_SERVICE_ROLE"
+os.environ["S3_BUCKET_DATA_LAKE"] = "test_datalake_bucket"
```

Comment:
Is this justed mocked or a real bucket.
In case you need one, there is `gfw-data-lake-test`

Reply:
it's just mocked